features/authorization/deny.feature

```diff
@@ -211,6 +211,79 @@ Feature: Authorization checking
     And the response should contain "ownerOnlyProperty"
     And the JSON node "ownerOnlyProperty" should be equal to the string "updated"
 
+  Scenario: An non existing entity should return Not found
```

Comment:
can you put a tag `@link_security` on this, and then inside `behat.yml.dist` in the `legacy` suite: 

```
      filters:
        tags: '~@postgres&&~@mongodb&&~@elasticsearch&&~@link_security'

```
thanks